packages/app/src/state/staking/actions.ts

```diff
@@ -108,6 +142,35 @@ export const fetchEscrowData = createAsyncThunk<
 	}
 })
 
+export const fetchEscrowV2Data = createAsyncThunk<
+	{ escrowData: EscrowData<string>[]; totalVestable: string },
+	void,
+	ThunkConfig
+>('staking/fetchEscrowV2Data', async (_, { extra: { sdk } }) => {
+	const { escrowData, totalVestable } = await sdk.kwentaToken.getEscrowV2Data()
+
+	return {
```

Comment:
same here for handling error and notifying

Reply:
Thanks. Updated.